packages/notifications-library/src/notifications_library/templates/base.html

```diff
@@ -5,28 +5,102 @@
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
 <title>{% block title %}{% endblock %}</title>
 <style>
-    body {
-        font-family: Arial, sans-serif;
-        margin: 0;
-        padding: 20px;
-        color: #333;
-    }
-    .container {
-        background-color: #f9f9f9;
-        padding: 20px;
-        border-radius: 5px;
-        box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
-    }
-    a {
-        color: #007bff;
-        text-decoration: none;
-    }
+  body {
+      font-family: Manrope, sans-serif;
+      margin: 0;
+      padding: 0;
+      background-color: #f4f4f4;
+  }
+  .email-container {
+      max-width: 600px;
+      margin: 20px auto;
+      background-color: #ffffff;
+      padding: 20px;
+      border-radius: 8px;
+      box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+  }
+  .header {
+      display: flex;
+      align-items: center;
+      border-bottom: 2px solid #ddd;
+      padding-bottom: 10px;
+      margin-bottom: 20px;
+  }
+  .logo {
+      height: 40px;
+  }
+  .content {
+      color: #333;
+      font-size: 16px;
+      line-height: 1.5;
+  }
+  a {
+      color: #007bff;
+      text-decoration: none;
+  }
+  .strong-button {
+      cursor: pointer;
+      background-color: {{ product.product_ui.strong_color }};
```

Comment:
HOw does this resolve if product.product_ui.strong_colo does not exists. E.g. assume we do not have a product.vendor that provides this value.

In this case, I guess there could be a default background but in other cases we should perhaps avoid to even set the value (e.g. with images, we might decided to have an alternative text instead)

Reply:
Added some defaults 👍